language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/DamlLedgerClient.java

```diff
@@ -68,14 +68,14 @@ public Builder withAccessToken(@NonNull String accessToken) {
       return this;
     }
 
-    public Builder withDeadline(@NonNull Deadline deadline) {
-      this.deadline = Optional.of(deadline);
+    public Builder withInterceptors(ClientInterceptor... interceptors) {
+      this.interceptors = interceptors;
```

Comment:
Two notes:
1. `withDeadline` has been shipped as part of a stable release (1.16) and we can't just drop it
2. In general I think client interceptors are a good idea but I would prefer to have a bit of design work happen before committing to something in that regard

Reply:
1. Looks like `withDeadline` hasn't been shipped in 1.16.
2. Sounds good.